packages/core/src/createStore.ts

```diff
@@ -68,19 +57,20 @@ export function createStore(
     }
   }
 
-  function _getState(target?: Atom<any>) {
+  function _getState(): State
+  function _getState<T>(target?: Atom<T>): State | T | undefined {
```

Comment:
_getState not retruns `undefined`

Reply:
if so we need change type of `getState` from `declareAtom.ts`